ewlegal.cls

```diff
@@ -0,0 +1,84 @@
+%% TODO Stock beginning text, copyrights and project info and all that
+
+%% Work consists of the files engwales-legal.cls and accompanying engwales-*.sty
+
+
+\NeedsTeXFormat{LaTeX2e}
+\ProvidesClass{ewlegal}[2021-02-21 Legal Templates for England and Wales]
+
+%% Uses article as a base
+\LoadClass[12pt,a4paper]{article}% TODO Perhaps use KOMA-Script?
+
+
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%% Options
+%%%%%
+
+%% Easy option management
+\RequirePackage{kvoptions}
+
+%% Select the template used
+% TODO Be able to nicely handle multiple templates being selected
+
+\DeclareBoolOption[false]{pleading}
+
```

Comment:
Nit: probably want these grouped so remove blank line here.

Reply:
Fixed, somewhat... Still a bit spaced out.

Reply:
> Still a bit spaced out
what do you mean?

Reply:
There was some space around them, but not between, but they should be okay now. Resolving for now.